core/server/api/v2/utils/serializers/input/posts.js

```diff
@@ -2,6 +2,14 @@ const _ = require('lodash');
 const debug = require('ghost-ignition').debug('api:v2:utils:serializers:input:posts');
 const url = require('./utils/url');
 
+function removeMobiledocFormat(frame) {
+    if (_.get(frame, 'options.formats') && _.get(frame, 'options.formats').includes('mobiledoc')) {
+        frame.options.formats = frame.options.formats.filter((format) => {
+            return (format !== 'mobiledoc');
+        });
```

Comment:
One unit test would be cool to avoid breaking the behaviour in case we change something in this file 👍